code/datums/weather/weather.dm

```diff
@@ -181,9 +181,16 @@
  */
 /datum/weather/proc/can_weather_act(mob/living/L)
```

Comment:
better var name, please

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names